content/en/code_analysis/static_analysis/_index.md

```diff
@@ -108,7 +108,30 @@ The two types of fixes are distinguished visually in the UI with different label
 {{< img src="code_analysis/static_analysis/static-analysis-ai-fix.png" alt="Visual indicator of an AI static analysis suggested fix" style="width:60%;">}}
 
 ### Ignoring violations
-To ignore a rule globally or by path(s), see [Static Analysis Setup][5]. 
+
+#### Ignore for a repository
+Add an ignore rule in your `static-analysis.datadog.yml` file. This example below ignores the rule `javascript-express/reduce-server-fingerprinting` for all directories.
+
+```
+rulesets:
+  - javascript-express
+    rules:
+      reduce-server-fingerprinting:
+        ignore: "**"
+```
+
+#### Ignore for a file or directory
+Add an ignore rule in your `static-analysis.datadog.yml` file. The example below ignores the rule `javascript-express/reduce-server-fingerprinting` for this file. For more information on how to ignore by path, see [Static Analysis Setup][5]. You can adapt this to use a path (or see our documentation about our configuration file).
```

Comment:
```suggestion
Add an ignore rule in your `static-analysis.datadog.yml` file. The example below ignores the rule `javascript-express/reduce-server-fingerprinting` for this file. For more information on how to ignore by path, see [Static Analysis Setup][5]. You can adapt this to use a path (or see the configuration file documentation).
```

Reply:
Minor update per the style guide here. Would it also be possible to provide a link to the configuration file documentation?

Reply:
The last sentence is repetitive, so we can remove it. Path prefixes are covered in the Setup doc page :)